package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "asset-pipe-build-server",
-  "version": "1.0.0-beta.13",
+  "version": "0.0.0-development",
```

Comment:
Given you are resetting version here, maybe we should rename project to be @asset-pipe/build-server instead :)?. I think so.

Reply:
The first published version under semantic-release will end up being 1.0.0 which seems fine to me in terms of continuity. It does seem redundant to have asset-pipe/asset-pipe-build-server but wouldn't we need to publish under an org? `@asset-pipe/build-server` since `build-server` won't be available on npm. Or are you thinking the package.json name stays `asset-pipe-build-server` and we publish under that but that the github name changes to just `build-server`?